Real_Masters_all/alumasso.xml

```diff
@@ -15860,7 +15860,7 @@
                 <container type="box" label="Box">162</container>
                 <unittitle><title render="italic">Preserving Our Nation's Heritage</title>, #533, #534, <unitdate type="inclusive" normal="1985-11-20">11/20/1985</unitdate></unittitle>
                 <physdesc>
-                  <extent>2 tapes</extent>
+                  <extent>2 audioassettes</extent>
```

Comment:
Nice catch! Should be fixed now.

On Wed, Sep 23, 2015 at 5:04 PM, Max Eckard <notifications@github.com>
wrote:

> In Real_Masters_all/alumasso.xml
> <https://github.com/bentley-historical-library/vandura/pull/231#discussion_r40257622>
> :
>
> > @@ -15860,7 +15860,7 @@
> >                  <container type="box" label="Box">162</container>
> >                  <unittitle><title render="italic">Preserving Our Nation's Heritage</title>, #533, #534, <unitdate type="inclusive" normal="1985-11-20">11/20/1985</unitdate></unittitle>
> >                  <physdesc>
> > -                  <extent>2 tapes</extent>
> > +                  <extent>2 audioassettes</extent>
>
> audiocassettes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bentley-historical-library/vandura/pull/231/files#r40257622>
> .
>